skyportal/models.py

```diff
@@ -647,6 +649,15 @@ class Obj(Base, ha.Point):
         doc="Comments posted about the object.",
     )
 
+    comments_on_spectra = relationship(
+        'CommentOnSpectrum',
+        back_populates='obj',
+        cascade='save-update, merge, refresh-expire, expunge, delete',
```

Comment:
You've already got the ondelete cascade set on the obj_id foreign key on the child class, so this should probably just be 
```suggestion
        cascade='save-update, merge, refresh-expire, expunge',
```